Index: linuxutils_2_26_01_02/packages/ti/sdo/linuxutils/sdma/src/module/sdmak.c
===================================================================
--- linuxutils_2_26_01_02.orig/packages/ti/sdo/linuxutils/sdma/src/module/sdmak.c
+++ linuxutils_2_26_01_02/packages/ti/sdo/linuxutils/sdma/src/module/sdmak.c
@@ -44,7 +44,7 @@
 #include <mach/hardware.h>
 #include <mach/dma.h>
 #include <mach/tc.h>
-#else
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(3,7,0)
 #include <plat/hardware.h>
 #include <plat/dma.h>
 #include <plat/tc.h>
@@ -97,6 +97,11 @@
 
 #endif /* LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,18) */
 
+#ifndef OMAP_DMA_NO_DEVICE
+#include <linux/omap-dma.h>
+#define OMAP_DMA_NO_DEVICE	0
+#endif
+
 #include "../interface/sdma.h"
 
 struct channel {
