From fed1d4701b8442c613e7ebc0bb94b251844d3b1c Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Fri, 10 Nov 2017 20:16:20 -0500
Subject: [PATCH] elf32-tic6x.c: fix implicit fallthrough error in gcc7

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 gdbc6x/bfd/elf32-tic6x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdbc6x/bfd/elf32-tic6x.c b/gdbc6x/bfd/elf32-tic6x.c
index fe82c8f..b046566 100644
--- a/gdbc6x/bfd/elf32-tic6x.c
+++ b/gdbc6x/bfd/elf32-tic6x.c
@@ -2365,7 +2365,7 @@ elf32_tic6x_relocate_section (bfd *output_bfd,
 		  goto done_reloc;
 		}
 	    }
-
+	    /* FALLTHRU */
 	case R_C6000_PCR_S12:
 	case R_C6000_PCR_S10:
 	case R_C6000_PCR_S7:
-- 
2.7.4

