From b77515953ba72728a10a46381d8b1dd73b89825d Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Fri, 10 Nov 2017 20:19:51 -0500
Subject: [PATCH] elf.c: correct fallthrough comment to recognize by gcc7

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 gdbc6x/bfd/elf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gdbc6x/bfd/elf.c b/gdbc6x/bfd/elf.c
index ede51e9..80bac08 100644
--- a/gdbc6x/bfd/elf.c
+++ b/gdbc6x/bfd/elf.c
@@ -1595,7 +1595,7 @@ bfd_section_from_shdr (bfd *abfd, unsigned int shindex)
 	      if (hdr->sh_link == (SHN_LORESERVE & 0xffff) /* SHN_BEFORE */
 		  || hdr->sh_link == ((SHN_LORESERVE + 1) & 0xffff) /* SHN_AFTER */)
 		break;
-	      /* Otherwise fall through.  */
+	      /* Else fall through.  */
 	    default:
 	      return FALSE;
 	    }
-- 
2.7.4

