From e4708813cc0d08b75a8db49dcde0722f3d3add5c Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Fri, 10 Nov 2017 19:30:22 -0500
Subject: [PATCH] reloc.c: add comments for implicit fallthrough error in gcc7

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 gdbc6x/bfd/reloc.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/gdbc6x/bfd/reloc.c b/gdbc6x/bfd/reloc.c
index 0d191f1..a6e0e12 100644
--- a/gdbc6x/bfd/reloc.c
+++ b/gdbc6x/bfd/reloc.c
@@ -7254,15 +7254,20 @@ bfd_default_reloc_type_lookup (bfd *abfd, bfd_reloc_code_real_type code)
 	{
 	case 64:
 	  BFD_FAIL ();
+	  /* FALLTHRU */
 	case 32:
 	  return &bfd_howto_32;
 	case 16:
 	  BFD_FAIL ();
+	  /* FALLTHRU */
 	default:
 	  BFD_FAIL ();
+	  /* FALLTHRU */
 	}
+	/* FALLTHRU */
     default:
       BFD_FAIL ();
+      /* FALLTHRU */
     }
   return NULL;
 }
-- 
2.7.4

