From 9e1b0f0d7148524c226befdef90849f9e143ef66 Mon Sep 17 00:00:00 2001
From: Denys Dmytriyenko <denys@ti.com>
Date: Fri, 17 May 2019 18:20:21 +0000
Subject: [PATCH] selftests: add "ptp" to TARGETS

Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 tools/testing/selftests/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/testing/selftests/Makefile b/tools/testing/selftests/Makefile
index 831022b1..6bd7cf81 100644
--- a/tools/testing/selftests/Makefile
+++ b/tools/testing/selftests/Makefile
@@ -20,6 +20,7 @@ TARGETS += net
 TARGETS += nsfs
 TARGETS += powerpc
 TARGETS += pstore
+TARGETS += ptp
 TARGETS += ptrace
 TARGETS += seccomp
 TARGETS += sigaltstack
-- 
2.17.1

