diff --git a/makefile_armv7 b/makefile_armv7
index a1fd5c0..d050ace 100644
--- a/makefile_armv7
+++ b/makefile_armv7
@@ -117,7 +117,7 @@ install:
 	install -d $(INSTALL_LIB_BASE_DIR)
 	$(CP) ./*.h                $(INSTALL_INC_BASE_DIR)/ti/drv/qmss
 	$(CP) ./include/*.h        $(INSTALL_INC_BASE_DIR)/ti/drv/qmss/include
-	$(CP) ./firmware/*.h       $(INSTALL_INC_BASE_DIR)/ti/drv/qmss/firmware
+	$(CP) -r ./firmware/*      $(INSTALL_INC_BASE_DIR)/ti/drv/qmss/firmware
 	$(CP) -r ./device $(INSTALL_INC_BASE_DIR)/ti/drv/qmss/
 	@$(CP) -r $(ARMV7LIBDIR)/*.a         $(INSTALL_LIB_BASE_DIR)
 	@$(CP) -r $(ARMV7LIBDIR)/*.so        $(INSTALL_LIB_BASE_DIR)
